src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        return Color.values()[random.nextInt(Color.values().length)].toString();
```

Comment:
Better to extract `random.nextInt(Color.values().length)` in local variable to make your code more readable.